Configs/supportbot.yml

```diff
@@ -47,7 +47,6 @@ WarningColour: "#fca117"
 # -------------------------
 
 InfoTitle: "Information"
-AboutMe: "Hi, I'm SupportBot an advanced discord ticket bot with 100% customization! [Download](https://emeraldsrv.dev/download)"
```

Comment:
Remove this due it being part of the description of a bot, set in the developer portal.